### PR TITLE
Update example console template for node exporter 0.16.0

### PR DESCRIPTION
**Pull request by brian-brazil**
@SuperQ

### CODE REVIEW

File: consoles/node-disk.html [hunk 1]
```diff
@@ -9,7 +9,7 @@ <h3>Disk I/O Utilization</h3>
   new PromConsole.Graph({
     node: document.querySelector("#diskioGraph"),
     expr: [
-      "irate(node_disk_io_time_ms{job='node',instance='{{ .Params.instance }}',device!~'^(md\\\\d+$|dm-)'}[5m]) / 1000 * 100",
+      "irate(node_disk_io_time_seconds_total{job='node',instance='{{ .Params.instance }}',device!~'^(md\\\\d+$|dm-)'}[5m]) / 1000 * 100",
```
Comment (SuperQ) [inline consoles/node-disk.html]: We can eliminate the `/ 1000` here.

File: consoles/node-disk.html [hunk 2]
```diff
@@ -38,34 +38,34 @@ <h3>Filesystem Usage</h3>
 {{ template "prom_right_table_head" }}
   <th colspan="2">Disks</th>
 </tr>
-{{ range printf "node_disk_io_time_ms{job='node',instance='%s'}" .Params.instance | query | sortByLabel "device" }}
+{{ range printf "node_disk_io_time_seconds_total{job='node',instance='%s'}" .Params.instance | query | sortByLabel "device" }}
   <th colspan="2">{{ .Labels.device }}</th>
   <tr>
     <td>Utilization</td>
-    <td>{{ template "prom_query_drilldown" (args (printf "irate(node_disk_io_time_ms{job='node',instance='%s',device='%s'}[5m]) / 1000 * 100" .Labels.instance .Labels.device) "%" "printf.1f") }}</td>
+    <td>{{ template "prom_query_drilldown" (args (printf "irate(node_disk_io_time_seconds_total{job='node',instance='%s',device='%s'}[5m]) / 1000 * 100" .Labels.instance .Labels.device) "%" "printf.1f") }}</td>
```
Comment (SuperQ) [inline consoles/node-disk.html]: Same here.

File: consoles/node-disk.html [hunk 3]
```diff
@@ -38,34 +38,34 @@ <h3>Filesystem Usage</h3>
...
 </tr>
 {{ define "roughlyNearZero" }}
 {{ if gt .1 . }}~0{{ else }}{{ printf "%.1f" . }}{{ end }}
 {{ end }}
-{{ range printf "node_filesystem_size{job='node',instance='%s'}" .Params.instance | query | sortByLabel "mountpoint" }}
+{{ range printf "node_filesystem_size_bytes{job='node',instance='%s'}" .Params.instance | query | sortByLabel "mountpoint" }}
   <tr>
     <td>{{ .Labels.mountpoint }}</td>
-    <td>{{ template "prom_query_drilldown" (args (printf "100 - node_filesystem_free{job='node',instance='%s',mountpoint='%s'} / node_filesystem_size{job='node'} * 100" .Labels.instance .Labels.mountpoint) "%" "roughlyNearZero") }}</td>
+    <td>{{ template "prom_query_drilldown" (args (printf "100 - node_filesystem_free_bytes{job='node',instance='%s',mountpoint='%s'} / node_filesystem_size_bytes{job='node'} * 100" .Labels.instance .Labels.mountpoint) "%" "roughlyNearZero") }}</td>
```
Comment (SuperQ) [inline consoles/node-disk.html]: should we change this from `free` to `avail`?  I find that most users need to know `avail`, as this is when things break down.

File: consoles/node-overview.html [hunk 4]
```diff
@@ -23,7 +23,7 @@ <h3>Disk I/O Utilization</h3>
   new PromConsole.Graph({
     node: document.querySelector("#diskioGraph"),
     expr: [
-      "irate(node_disk_io_time_ms{job='node',instance='{{ .Params.instance }}',device!~'^(md\\\\d+$|dm-)'}[5m]) / 1000 * 100",
+      "irate(node_disk_io_time_seconds_total{job='node',instance='{{ .Params.instance }}',device!~'^(md\\\\d+$|dm-)'}[5m]) / 1000 * 100",
```
Comment (SuperQ) [inline consoles/node-overview.html]: Same `/ 1000` here.

File: consoles/node-overview.html [hunk 5]
```diff
@@ -59,47 +59,47 @@ <h3>Memory</h3>
...
 </tr>
 <tr>
   <th colspan="2">Disks</th>
 </tr>
-{{ range printf "node_disk_io_time_ms{job='node',instance='%s',device!~'^(md\\\\d+$|dm-)'}" .Params.instance | query | sortByLabel "device" }}
+{{ range printf "node_disk_io_time_seconds_total{job='node',instance='%s',device!~'^(md\\\\d+$|dm-)'}" .Params.instance | query | sortByLabel "device" }}
   <tr>
     <td>{{ .Labels.device }} Utilization</td>
-    <td>{{ template "prom_query_drilldown" (args (printf "irate(node_disk_io_time_ms{job='node',instance='%s',device='%s'}[5m]) / 1000 * 100" .Labels.instance .Labels.device) "%" "printf.1f") }}</td>
+    <td>{{ template "prom_query_drilldown" (args (printf "irate(node_disk_io_time_seconds_total{job='node',instance='%s',device='%s'}[5m]) / 1000 * 100" .Labels.instance .Labels.device) "%" "printf.1f") }}</td>
```
Comment (SuperQ) [inline consoles/node-overview.html]: More `/ 1000`

File: consoles/node-disk.html (hunk 3 above)
Comment (brian-brazil) [reply consoles/node-disk.html]: Makes sense.

**Comment by brian-brazil** [comment]
Thanks for spotting those, PTAL.

**Comment by xaka** [comment]
I see the fix landed on Jun 8 while latest release (2.3.2) happened on Jul 12 without it. Any plans on including it in upcoming 2.x? 

Thanks!

**Comment by SuperQ** [comment]
Yes, we're in the process of starting a new release schedule soon.

You can always copy the existing templates out of master, they're not really Prometheus version dependent.

**Pull request merged**